### ISSUE TITLE
Is it possible to 'reprocess' audiobooks after downloading them?

### ISSUE THREAD

**Issue by walkswithpies** [rmcrackan/Libation#835]
I would like to reprocess some audio books with some of the optional libation file settings turned on (such as fixing up the meta data and moving the moov block).  I asked Libation to keep the original aax files, which it did.  I had a quick Look at the CLI but nothing jumped out at me.  

If I ask Libation to re-download the books will it realise I already have a stored local download and go from there?  

Thanks for this amazing tool!

**Comment by rmcrackan** [comment]
After you download a file, Libation doesn't care what you do with it. We just track the fact that it was downloaded. Similarly, when you download again it doesn't care whether or not an old file exists -- it'll just do its job and download your book.

To re-download a book, rt-clk to change it's status to 'not downloaded' and do a new download. (It sounds like you already had that part; just being clear esp for future readers.)

Your new settings will be used for your next download. If there's ever an existing file with the exact same name, we'll do that ` (1)` thing at the end

Let me know if you have any other questions

**Comment by walkswithpies** [comment]
Thanks for the reply.

I hope you don’t mind me emailing but the GitHub thread was closed.  I was hoping to avoid having to download 150Gb of books again (which are already on my drive in aax format) in order to convert them to m4b again - but this time with a couple of different Libation options.  I suppose I could write a shell script that loops through them invoking ffmpeg, but I don’t have enough knowledge to know which ffmpeg parameters (etc) to use to accomplish everything Libation does.

Which means I am asking for a Libation option to use an existing aax file I suppose! 

I understood everything you said though and want to say thanks for making Libation and sharing it with us :)

Best wishes,
Steven

**Comment by rmcrackan** [comment]
I don't mind continuing the discussion -- I still get notifications of replies on closed threads.

Libation can't convert just from aax; it all starts from downloading from audible. If your files are actually aax (and not aaxc) then [inAudible](https://github.com/rmcrackan/inAudible) is a choice for local-only conversion. Or as you said: scripting ffmpeg. ffmpeg is an amazing swiss army knife but dear god is the interface abysmal. Note: I didn't make inAudible; I just got permission from the author to host the installer.

On a high speed connection, downloading with Libation is super fast. We do the decryption on the fly so it's not even a 2 step process any more (we used to download first, then decrypt)
